srv/salt/_modules/fs.py

```diff
@@ -210,9 +210,8 @@ def _add_fstab_entry(_uuid, path, fstype, subvol):
     try:
         with open('/etc/fstab', 'r') as _fstab:
             fstab_entries = [line.rstrip('\n') for line in _fstab]
-    # pylint: disable=bare-except
-    except:
-        log.error("Failed to read /etc/fstab.")
+    except (IOError, FileNotFoundError):
```

Comment:
I went to check what `open` raises since I didn't know it off the top of my head and found out that nowadays it is `OSError` instead of `IOError`. `IOError` is an alias to `OSError`, so that still works, but the new one is `OSError`. I don't know if python will break the backwards compability between the two in the future, I think we are on the safer side if we go with `OSError`

Reply:
Good point. 
How did you get an OSError?

when i do it in both versions of python i get this: http://paste.opensuse.org/48894588

Reply:
I did not try it out, I just read the docs at https://docs.python.org/3.5/library/functions.html#open (I wanted to link this earlier, but forgot it).

After reading your paste I tried to get an OSError in python3, but could not do it either.

Reply:
Ah, true. In py3 the FileNotFoundError is a subclass of the OSError, but i would rather stay with a more specific exception.
I included the IOError only for py2, bacause back then the exception dependencies were a mess.